src/mlpack/core/data/load.hpp

```diff
@@ -114,49 +115,49 @@ bool Load(const std::string& filename,
  *
  * @cond
  */
-
+/*
 extern template bool Load<int>(const std::string&,
                                arma::Mat<int>&,
                                const bool,
                                const bool,
-                               const arma::file_type);
+                               const file_type);
 
 // size_t and uword should be one of these three typedefs.
 extern template bool Load<unsigned int>(const std::string&,
                                         arma::Mat<unsigned int>&,
                                         const bool,
                                         const bool,
-                                        const arma::file_type);
+                                        const file_type);
 
 extern template bool Load<unsigned long>(const std::string&,
                                          arma::Mat<unsigned long>&,
                                          const bool,
                                          const bool,
-                                         const arma::file_type);
+                                         const file_type);
 
```

Comment:
It will be easier to remove them rather than adding comment, this will improve the review process :+1: